03 Tests samples/README.md

```diff
@@ -0,0 +1,30 @@
+# Testing samples for an app using redux architecture and hooks for some components.
+
+## In this folder you can see tests samples for different test types.
+
+- **Redux architecture** test samples:
+
+  - Action test files samples: [MessageActionTest](./src/redux/actions/MessagesActions.test.tsx)
+  - Reducers test files: [MessageReducerTest](./src/redux/reducers/Mesages.Reducer.test.tsx)
+
+- **API** test sample: [ApiTest](./src/api/myApi.test.tsx)
+
+- **Components** test samples:
+   - Message form test: [FormComponent](./src/components/MessageForm/MessageForm.test.tsx)
+   - Message list test: [ListComponent](./src/components/MessageList/MessageList.test.tsx)
+   - Message section test: [SectionComponent](./src/components/MessagesSection/MessagesSection.test.tsx)
+   
+- **Model-mappers** test sample:
+
+- Tests samples for events onChange ( simulate, fireEvent, ...) (Regina)
+- Tests samples for events onClick (Aurora)
+- Tests samples for events onScroll (Daily)
+- Test samples for onMouseUp and onMouseDown (Regina)
+- Test samples for onContextMenu (Regina)
+- Test samples for onSubmit (Aurora)
+
+- Tests samples for useSuscribeToScroll (Daily)
+- Tests samples using ref an useRef hook (Regina, Daily)
```

Comment:
What about removing the names here once the issues have been created for each person?

Reply:
Yes, done, removed now, thanks!

Reply:
> What about removing the names here once the issues have been created for each person?

Yes, done, removed now, thanks!